sa/sa.go

```diff
@@ -1367,6 +1367,9 @@ func AuthzMapToPB(m map[string]*core.Authorization) (*sapb.Authorizations, error
 // objects if the V2 field is set. This method is intended to deprecate AddPendingAuthorizations
 func (ssa *SQLStorageAuthority) NewAuthorizations2(ctx context.Context, req *sapb.AddPendingAuthorizationsRequest) (*sapb.Authorization2IDs, error) {
 	ids := &sapb.Authorization2IDs{}
+	var vals []interface{}
+	var qs strings.Builder
```

Comment:
- Nit, presume `qs` is short for `queryString`; I think it would probably be more accurate to call it a `queryBuff`
- Nit, I think `vals` could be more descriptive as `queryArgs` since that's where it ends up

```suggestion
	var queryArgs []interface{}
	var queryBuff strings.Builder
```

Reply:
OK